app/assets/styles/styles.css

```diff
@@ -170,3 +170,26 @@ a {
     font-size: 1.25rem;
   }
 }
+
+.donate {
+  height: 300px;
+  font-size: 1.5rem;
+  margin-left: 2em;
+  margin-right: 2em;
+  background-image: linear-gradient(rgba(21, 21, 78, 0.5), rgba(21, 21, 78, 0.6)), url("/home/salvia/news.macile.org/app/assets/images/salvia-team.webp");
```

Comment:
The image path needs to be absolute from the project directory not your computer directory